apps/authentication-service/.eslintrc.cjs

```diff
@@ -29,4 +29,14 @@ module.exports = {
     project: true,
     tsconfigRootDir: __dirname,
   },
+  rules: {
+    "@typescript-eslint/no-unused-vars": [
+      "error",
+      {
+        argsIgnorePattern: "^_",
```

Comment:
Why would we ignore based on _?

Reply:
For express middleware, you have to have all the parameters, or it changes how the function works. You can see an example of this in the `.src/middleware/not-found.middleware.ts` file. 

If I were to remove the `_request`, and `_next` params, which aren't used, it would alter now express views the function. 

If I don't have this rule, then I would have to put ignore rules in the file instead. I thought this was cleaner, and is a fairly common pattern.

Reply:
@wesrisenmay-mx Let me know if you have any additional feedback.